extensions/cache/src/main/resources/benchmark-redis.xml

```diff
@@ -1,68 +1,73 @@
+<!-- This test shows example comparison between three products -->
 <benchmark xmlns="urn:radargun:benchmark:3.0">
-
    <master bindAddress="${master.address:127.0.0.1}" port="${master.port:2103}"/>
 
    <clusters>
-      <cluster size="8">
-         <group name="server" size="6" />
-         <group name="client" size="2" />
+      <cluster size="3">
+         <group name="client" size="3"/>
       </cluster>
    </clusters>
 
    <configurations>
       <config name="redis">
-         <setup group="server" plugin="redis32">
```

Comment:
Yep, the example benchmarks should be self-contained. I think that since @Holmistr was using some Enterprise version, he could not add that one to this configuration. I don't think we should call the plugin itself `Enterprise` if it's using community client to connect to `Enterprise` server, though, but I might misunderstood.